src/test/util_tests.cpp

```diff
@@ -765,7 +765,7 @@ BOOST_AUTO_TEST_CASE(test_LockDirectory)
     // Probing lock from our side now should succeed, but not hold on to the lock.
     BOOST_CHECK_EQUAL(LockDirectory(dirname, lockname, true), true);
 
-    // Try to acquire the lock in the child process, this should be succesful.
+    // Try to acquire the lock in the child process, this should be successful.
```

Comment:
Already reported in #12543.

Reply:
So should I revert this one or just do nothing?

Reply:
Revert or rebase on top of `master` now that #12543 is merged.

Reply:
Reverted. Not familiar with git rebase and such...